lib/modules/@apostrophecms/schemas/src/apos/components/AposSchema.vue

```diff
@@ -38,11 +38,11 @@ export default {
   emits: ['input'],
   data() {
     return {
-      // TODO: Complete this with other schema field types.
-      fieldComponentMap: {
-        string: 'AposInputString',
-        boolean: 'AposInputBoolean'
-      }
+      piece: {
```

Comment:
It could be a widget schema as well, so it wouldn't be a piece, or even a full document. Though `data` seems too generic. Maybe `docData`?

Reply:
I don't think it is too generic, as schemas can be used to model things that aren't docs (widgets, or things we're not anticipating at all right now).